### PR TITLE
Document HTML export dialog

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Related with https://github.com/omichel/webots/pull/6061.

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by omichel** [comment]
:eyes:

File: guide/web-streaming.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ Use the buttons on the top of the page to connect to the Webots streaming server
 `IP` address and port for the streaming server and the client page should match.
 The port should not be used by another application.
 
+The streaming functionalities will use the `X3D` export parameters stored in the [project file](the-standard-file-hierarchy-of-a-project.md#the-project-files).
+This parameters are set when [exporting a HTML5 model](web-scene.md#how-to-export-a-web-scene) or [recording an animation](web-animation.md#how-to-export-a-web-animation).
```
Comment (tn12787) [inline guide/web-streaming.md]: This parameters -> *These* parameters

File: guide/web-streaming.md [hunk 2]
```diff
@@ -47,6 +47,9 @@ Use the buttons on the top of the page to connect to the Webots streaming server
 `IP` address and port for the streaming server and the client page should match.
 The port should not be used by another application.
 
+The streaming functionalities will use the `X3D` export parameters stored in the [project file](the-standard-file-hierarchy-of-a-project.md#the-project-files).
```
Comment (tn12787) [inline guide/web-streaming.md]: I'd remove the "will" here, the present tense is better here in my opinion.

Comment (omichel) [inline guide/web-streaming.md]: I would use the singular form for "functionalities".

File: guide/web-streaming.md (hunk 1 above)
Comment (omichel) [inline guide/web-streaming.md]: This &rarr; These

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**